backend/LeetWars.Core/LeetWars.Core.BLL/Services/ChallengeService.cs

```diff
@@ -78,6 +78,12 @@ IUserGetter userGetter
                     filterTags.All(tag => challenge.Tags.Contains(tag)));
             }
 
+            if(filters.DifficultyLevel is not null)
+            {
+                challenges = challenges.Where(challenge => 
+                challenge.Level != null && challenge.Level.Name.Equals(filters.DifficultyLevel));
```

Comment:
```suggestion
                challenge.Level is not null && challenge.Level.Name.Equals(filters.DifficultyLevel));
```

Reply:
Can't do, compiler does not allow

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/57751436/6270e075-dfb8-4ee1-aec1-1441e36d9443)